web/src/invite.ts

```diff
@@ -413,14 +447,22 @@ function open_invite_user_modal(e: JQuery.ClickEvent<Document, undefined>): void
             }
         });
 
-        $(".custom-expiration-time").on("change", () => {
+        $("#custom-invite-expiration-time").on("change", () => {
```

Comment:
This change needs to happen in the CSS change commit, because you elminate the "custom-expiration-time" class in that commit.

Reply:
Should this be `custom-expiration-time-input`? Usually I expect an on-change to be registered on an `input`, not `div`.

Reply:
I think both the time input and unit dropdown ids would need to be included in that follow-up.